src/Configurator/BundlesConfigurator.php

```diff
@@ -90,6 +90,14 @@ private function dump(string $file, iterable $bundles): void
         }
 
         file_put_contents($file, $contents);
+
+        if (function_exists('opcache_invalidate')) {
+            opcache_invalidate($file);
+        }
+
+        if (function_exists('apc_delete_file')) {
+            apc_delete_file($file);
```

Comment:
Are you sure that it is really not possible to install the extension with PHP 7? I did never try that and I also think that doing so is not a common use case. But I did not find any information on that it is not possible.